.changes/unreleased/NOTES-20250414-095537.yaml

```diff
@@ -0,0 +1,5 @@
+kind: NOTES
+body: 'This beta pre-release adds support for managed resource identity, which can be used with Terraform v1.12.0-beta1. Acceptance tests can use the `ImportBlockWithResourceIdentity` kind to exercise the import of a managed resource using its resource identity object values instead of using a string identifier.'
```

Comment:
😆 

```suggestion
body: 'This beta pre-release adds support for managed resource identity, which can be used with Terraform v1.12.0-beta2. Acceptance tests can use the `ImportBlockWithResourceIdentity` kind to exercise the import of a managed resource using its resource identity object values instead of using a string identifier.'
```